
{'date': 'Mon Jul 13 2020 19:21:45.251 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 39
  keys = dict(x=+1, y=-1):
                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jul 13 2020 19:22:00.778 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 39
  keys = dict(x=1, y=-1):
                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jul 13 2020 19:22:10.619 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 51
    Eventos().vai()
  module <module> line 29
    self.cacada.elt.bind("keypressed", self.anda_banhista)
AttributeError: 'Eventos' object has no attribute 'cacada'
'''},
{'date': 'Mon Jul 13 2020 19:35:24.835 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 41
  keys = dict(37=1, 39=-1)
                ^
SyntaxError: keyword can't be an expression
'''},
{'date': 'Mon Jul 13 2020 19:45:15.861 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  if 
      ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Jul 14 2020 12:24:58.865 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 38
    Tabuleiro().vai()
  module <module> line 23
    self.tabuleiro = Cena(self.MADEIRA)
  module _spy.vitollino.main line 1003
    self._cria_divs(width)
  module _spy.vitollino.main line 1008
    divesq.style.width = width // 3  # 100
TypeError: unsupported operand type(s) for //: 'set' and 'int'
'''},
{'date': 'Tue Jul 14 2020 12:32:01.244 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 38
    Tabuleiro().vai()
  module <module> line 24
    self.tabua = self.table(self.tabuleiro)
  module <module> line 34
    return [[casa(i, j) for i in range(linhas)] for j in range(colunas)]
  module <module> line 31
    t, margin = self.lado = self.lado // 3
TypeError: 'int' object is not iterable
'''},
{'date': 'Tue Jul 14 2020 13:10:40.703 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 41
  casa.elt <= = SPAN("", style={"fontSize":"30px"})
               ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Jul 14 2020 14:33:36.593 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    Tabuleiro().vai()
  module <module> line 34
    d = Elemento(self.dPIN, cena=self.tabuleiro, x=t*5, y=t*3, w=t*2, h=t*4)
AttributeError: 'Tabuleiro' object has no attribute 'dPIN'
'''},
{'date': 'Tue Jul 14 2020 15:19:45.724 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 53
  bx, by = -x*t, -y*t
                ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Jul 14 2020 15:27:12.812 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 54
  casa = Elemento(fundo, x=x, y=y, w=t, h=1.44*t, cena=base, tipo="auto"
                                                                                                                                                       ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Jul 14 2020 16:11:18.571 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 62
    Tabuleiro().vai()
  module <module> line 35
    self.pins(self.tabuleiro, self.LPIN)
  module <module> line 58
    return [[pin(i, j) for j in range(linhas)] for i in range(colunas)]
  module <module> line 56
    casa.elt.style.update({"background-position": f"{-bx}px {-by}px"})
AttributeError: 'CSSProperty' object has no attribute 'update'
'''},
{'date': 'Wed Jul 15 2020 10:09:50.956 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    Tabuleiro().vai()
  module <module> line 33
    self.tabua = self.table(self.tabuleiro, self.TABULEIRO, mx=margin, my=margin)
  module <module> line 48
    return [[Casa(i, j, dx, dy, mx, my) for j in range(linhas)] for i in range(colunas)]
  module <module> line 22
    super.__init__(fundo, x=x, y=y, w=t, h=t, cena=base)
TypeError: object() takes no parameters
'''},
{'date': 'Wed Jul 15 2020 10:12:06.243 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    Tabuleiro().vai()
  module <module> line 33
    self.tabua = self.table(self.tabuleiro, self.TABULEIRO, mx=margin, my=margin)
  module <module> line 48
    return [[Casa(i, j, dx, dy, mx, my) for j in range(linhas)] for i in range(colunas)]
  module <module> line 22
    super().__init__(fundo, x=x, y=y, w=t, h=t, cena=base)
  module _spy.vitollino.main line 541
    self.scorer = dict(ponto=1, valor=cena.nome, carta=tit or img, casa=self.xy, move=None)
AttributeError: 'int' object has no attribute 'nome'
'''},
{'date': 'Wed Jul 15 2020 10:13:42.939 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 48
  return [[Casa(, base, fundoi, j, dx, dy, mx, my) for j in range(linhas)] for i in range(colunas)]
                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 10:41:30.905 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  super().__init__(, w=dx, h=dy, cena=casa, tipo="auto"
                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 10:41:40.640 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  super().__init__(, w=dx, h=dy, cena=casa, tipo="auto",
                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 10:42:14.410 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 19
    class Pino(Elemento):
  module <module> line 28
    @static
NameError: name 'static' is not defined
'''},
{'date': 'Wed Jul 15 2020 10:42:37.225 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 19
    class Pino(Elemento):
  module <module> line 28
    @static_method
NameError: name 'static_method' is not defined
'''},
{'date': 'Wed Jul 15 2020 10:43:18.769 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 62
    Tabuleiro().vai()
  module <module> line 52
    self.pin_topo = Pino.pinos("topo", self.tab_topo)
  module <module> line 31
    return [Pino(indice, cor, casa) for indice, casa in casas]
  module <module> line 23
    pino, modulo = COR[cor], MOD[cor]
NameError: name 'COR' is not defined
'''},
{'date': 'Wed Jul 15 2020 10:45:59.539 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  style={"background-position"=f"{-x}px {-y}px"})
                              ^
SyntaxError: can't assign to function call 
'''},
{'date': 'Wed Jul 15 2020 11:02:10.943 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 53
  self.pin_lado = Pino.pinos("lado", self.tab_lado, dy=106
                                                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 11:48:15.561 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 29
  super().__init__(pino, x=8, y=-2, w=dx, h=dy, cena=casa, tipo="auto", tit=nome, drag=True
                                                                                                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 15 2020 11:58:06.295 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 78
    Tabuleiro().vai()
  module <module> line 63
    self.tabua = self.table(self.tabuleiro, self.TABULEIRO, mx=margin, my=margin)
  module <module> line 74
    return [[Casa(base, fundo, i, j, dx, dy, mx, my) for j in range(linhas)] for i in range(colunas)]
  module <module> line 48
    super().__init__(fundo, x=x, y=y, w=t, h=t, cena=base, drop=dropper)
  module _spy.vitollino.main line 551
    _ = self.entra(cena) if cena and (cena != INVENTARIO) else None
  module <module> line 51
    Pinos.pino(nome).entra(self)
NameError: name 'Pinos' is not defined
'''},
{'date': 'Wed Jul 15 2020 11:58:45.623 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 78
    Tabuleiro().vai()
  module <module> line 63
    self.tabua = self.table(self.tabuleiro, self.TABULEIRO, mx=margin, my=margin)
  module <module> line 74
    return [[Casa(base, fundo, i, j, dx, dy, mx, my) for j in range(linhas)] for i in range(colunas)]
  module <module> line 48
    super().__init__(fundo, x=x, y=y, w=t, h=t, cena=base, drop=dropper)
  module _spy.vitollino.main line 551
    _ = self.entra(cena) if cena and (cena != INVENTARIO) else None
  module <module> line 51
    Pino.pino(nome).entra(self)
  module <module> line 35
    return Pino.PINOS[nome]
KeyError: <_spy.vitollino.main.Cena object>
'''},
{'date': 'Wed Jul 15 2020 12:11:46.722 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 55
  , livre=False
   ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Jul 17 2020 10:57:17.999 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 11:21:53.752 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 11:23:12.160 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 11:28:35.636 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 11:31:10.105 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 11:31:51.141 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 11:49:47.784 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 11:50:52.472 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 11:56:42.378 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 11:57:45.168 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 12:03:15.789 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
TypeError: 'module' object is not callable
'''},
{'date': 'Fri Jul 17 2020 12:07:13.472 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 72
    Eventos().vai()
  module <module> line 36
    self.calcada = Cena(self.CENA_CALCADA, ditreita=Praia())
  module _spy.vitollino.main line 1001
    Cena.c(**kwargs)
  module _spy.vitollino.main line 1044
    imagem, kwargs = (imagem, {}) if isinstance(imagem, str) \
AttributeError: 'Praia' object has no attribute '__getitem__'
'''},
{'date': 'Sun Jul 26 2020 12:24:48.713 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 69
    class IntegerArithmeticTestCase(unittest.TestCase):
AttributeError: 'module' object has no attribute 'TestCase'
'''},